src/System.Net.Security/ref/System.Net.Security.cs

```diff
@@ -99,6 +99,7 @@ public partial class SslStream : AuthenticatedStream
         public override bool IsSigned { get { return default(bool); } }
         public virtual System.Security.Authentication.ExchangeAlgorithmType KeyExchangeAlgorithm { get { return default(System.Security.Authentication.ExchangeAlgorithmType); } }
         public virtual int KeyExchangeStrength { get { return default(int); } }
+        public virtual string NegotiatedApplicationProtocol { get { return default(string); } }
```

Comment:
I didn't realize this is adding new surface area.  That makes this a bigger deal.  Such surface area will need to go through the API review process before this PR can be considered further.